file-service-persistence/src/main/java/uk/gov/justice/services/fileservice/repository/ContentJdbcRepository.java

```diff
@@ -29,9 +29,9 @@
  */
 public class ContentJdbcRepository {
 
-    public static final String SQL_FIND_BY_FILE_ID = "SELECT content FROM content WHERE file_id = ? ";
-    public static final String SQL_INSERT_CONTENT = "INSERT INTO content(content, file_id) VALUES(?, ?)";
-    public static final String SQL_DELETE_CONTENT = "DELETE FROM content WHERE file_id = ? ";
+    public static final String SQL_FIND_BY_FILE_ID = "SELECT content, deleted FROM content WHERE file_id = ? ";
+    public static final String SQL_INSERT_CONTENT = "INSERT INTO content(file_id, content, deleted) VALUES(?, ?, ?)";
```

Comment:
could we have deleted as a default value in the liquibase?
then insert doesn't need to care but the update would.